kinto_wizard/yaml2kinto.py

```diff
@@ -143,5 +145,24 @@
                                                 data=record_data,
                                                 permissions=record_permissions)
 
+                if delete_missing_records and collection_records:
+                    # Fetch all records IDs
+                    file_records_ids = set(collection_records.keys())
+                    server_records_ids = set(current_collection['records'].keys())
+
+                    to_delete = server_records_ids - file_records_ids
+                    if not force:
+                        message = ("Are you sure that you want to delete the "
+                                   "following {} records?".format(len(list(to_delete))))
+                        value = input(message)
+                        print(to_delete)
```

Comment:
do we want this print?

Reply:
I figured it would give extra information about the records that are going to be deleted. But I could do it only in verbose mode maybe?